src/components/Questionnaire/QuestionTypes/MedicationRequestQuestion.tsx

```
@@ -1,7 +1,7 @@
 import { MinusCircledIcon } from "@radix-ui/react-icons";
 import { useQuery } from "@tanstack/react-query";
 import { t } from "i18next";
-import { ChevronsDownUp, ChevronsUpDown } from "lucide-react";
+import { X, ChevronsDownUp, ChevronsUpDown } from "lucide-react";
```

ESLint

[error] Replace `X,·ChevronsDownUp,·ChevronsUpDown` with `ChevronsDownUp,·ChevronsUpDown,·X`

(prettier/prettier)